### PR TITLE
added additional basic packages for atom, to help be productive :)

### PR DESCRIPTION
**Pull request by sharepointoscar**
What do you think of adding these basic package to Atom at deployment time? I like the terminal built-into Atom...  Or let me know what else you feel I can enhance this repo with, happy to help :)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (StefanScherer) [inline .vscode/settings.json:1]: Is this file needed in the pull request?

File: scripts/install-atom.ps1 [hunk 2]
```diff
@@ -4,3 +4,7 @@ $env:PATH="$env:PATH;$env:LOCALAPPDATA\atom\bin"
 apm install language-powershell
 apm install language-batch
 apm install language-docker
+apm install franken-berry-ui
+apm install franken-berry-syntax
+apm install file-icons
+apm install platformio-ide-terminal
```
Comment (StefanScherer) [inline scripts/install-atom.ps1:10]: This plugin looks very interesting

**Review by StefanScherer — COMMENTED** [review]
Thank you for the contribution.

This setup should be general purpose. Color themes are very personal and I don't want to end up installing tons of themes for everyone. Maybe we can add an additional script block to run further `apm install` commands that are not part of this Git repo.

File: scripts/install-atom.ps1 (hunk 2 above)
Comment (sharepointoscar) [reply scripts/install-atom.ps1:10]: Great idea, so how about I then create an additional .ps1 file call it something like **atom-add-ons.ps1** and include various `apm install` commands for items that are not part of the repo?

Makes sense on the themes and not installing them as they are in fact a personal preference.  I'll remove those.

File: .vscode/settings.json (hunk 1 above)
Comment (sharepointoscar) [reply .vscode/settings.json:1]: Oops not needed, I'll add it to the _.gitignore_  :)

File: scripts/install-atom.ps1 (hunk 2 above)
Comment (StefanScherer) [reply scripts/install-atom.ps1:10]: Sounds good.

**Comment by StefanScherer** [comment]
I recently switched the windows_2016_docker packer template to Windows Server Core to have the same "experience" as in 1709 or Insider builds. So I think if we should remove the editors from this provisioning. Or provide a UI variant starting from windows_2016 template that then has to install Containers feature and Docker.

**Comment by StefanScherer** [comment]
Closing this. Thanks for the idea, but I think the repo should give just a first start. Fork it and enhance it for your needs is always welcome :-)
